### PR TITLE
Fix/cran examples

### PR DESCRIPTION
**Pull request by sboysel**

### CODE REVIEW

**Comment by DavisVaughan** [comment]
Did they tell you to change dontrun to donttest? I've seen somewhere (dont know where) that CRAN eventually runs things in donttest anyways

File: R/fredr_series_observations.R [hunk 1]
```diff
@@ -116,21 +116,24 @@
...
 #' # To retrieve values for multiple series, use purrr's map_dfr() function.
-#' library(tidyverse)
-#' map_dfr(c("UNRATE", "OILPRICE"), fredr)
 #'
-#' # Using pmap_dfr() allows you to use varying optional parameters as well
-#' params <- list(
-#'   series_id = c("UNRATE", "OILPRICE"),
-#'   frequency = c("m", "q")
-#' )
+#' if (requireNamespace("purrr", quietly = TRUE)) {
```
[the remark this answers is not in the document]
Comment (DavisVaughan) [reply R/fredr_series_observations.R]: Oh i see. Makes sense!

[the remark this answers is not in the document]
Comment (sboysel) [reply R/fredr_series_observations.R]: It seems like you need to account for the member of `tidyverse` individually in examples.

[the remark this answers is not in the document]
Comment (DavisVaughan) [reply R/fredr_series_observations.R]: I just remembered this recent article from hadley and the crew on using `tidyverse` in packages. Long story short, not recommended 
https://www.tidyverse.org/articles/2018/06/tidyverse-not-for-packages/

[the remark this answers is not in the document]
Comment (sboysel) [reply R/fredr_series_observations.R]: OK I see.  I could remove it from Suggests.  Should be the only place it appears outside of vignettes.

No worries!

[the remark this answers is not in the document]
Comment (DavisVaughan) [reply R/fredr_series_observations.R]: Sorry, maybe I dont quite understand. 

Do we use `tidyverse` anywhere? If it is in the vignettes, and we use, say, `dplyr` in the vignettes, then we need to switch out `tidyverse` for just `dplyr` in Suggests

[the remark this answers is not in the document]
Comment (DavisVaughan) [reply R/fredr_series_observations.R]: A quick look reveals that we use `purrr` `dplyr` and `ggplot2` in 2 vignettes https://github.com/sboysel/fredr/search?q=tidyverse&unscoped_q=tidyverse

[the remark this answers is not in the document]
Comment (sboysel) [reply R/fredr_series_observations.R]: OK I think I forgot how exactly I had used it.  Thanks for tracking that down.

I can open a new PR explicitly adding `purrr`, `dplyr` and `ggplot2` to Suggests and `tidyverse`.

[the remark this answers is not in the document]
Comment (DavisVaughan) [reply R/fredr_series_observations.R]: Yes I think the steps are 

* Add `purrr` `dplyr` `ggplot2` to suggests
* Remove `tidyverse` from suggests
* replace `library(tidyverse)` with `library(purrr)` or whatever specific package / packages were needed

[the remark this answers is not in the document]
Comment (sboysel) [reply R/fredr_series_observations.R]: Addressing this in #65

**Pull request merged**